crates/turborepo-repository/src/change_mapper.rs

```diff
@@ -47,17 +95,9 @@ impl<'a> ChangeMapper<'a> {
         changed_files: HashSet<AnchoredSystemPathBuf>,
         lockfile_change: Option<LockfileChange>,
     ) -> Result<PackageChanges, ChangeMapError> {
-        let global_change =
-            self.repo_global_file_has_changed(&Self::DEFAULT_GLOBAL_DEPS, &changed_files)?;
-
-        if global_change {
-            return Ok(PackageChanges::All);
```

Comment:
No longer returning that all packages have changed on a root `package.json` alteration feels like something that needs more testing/validation.

Reply:
I removed this because it's redundant. If we return the root workspace as changed if *any* file not in a package is changed, then there's no reason to also have this code.